site/docs/how-tos/grpc.md

```diff
@@ -246,6 +246,15 @@ stage.mapUsingServiceAsyncBatched(bidiService,
 })
 ```
 
+If your batch takes close to or more than 1 s (including the network
+overhead) you should increase the value for following properties to
+perform clean shutdown:
+
+```
+jet.grpc.destroy.timeout
```

Comment:
maybe we should point the user to where you can learn more about these proeprties?